docs/operator-manual/Rancher-Fleet.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_label: "Rancher Fleet"
+title: ""
+---
+
+# Managing Kubewarden with Rancher Fleet
+
+The Kubewarden Helm charts, as other Helm charts, can be managed via [Rancher
+Fleet](https://fleet.rancher.io/). Rancher Fleet uses Kubernetes CRDs to define
+a GitOps approach for managing Kubernetes clusters. It does this by defining
+Fleet Bundles.
+
+The Kubewarden charts are normal charts, and they have defined dependencies that
+need to be ready before the dependent chart is deployed:
+- First, install `cert-manager`, as it is needed for the Kubewarden stack default config.
+- Second, install `kubewarden-crds`, as it is needed for the rest of Kubewarden charts.
+- Third, install `kubewarden-controller`, as it is needed by `kubewarden-defaults`.
+- Fourth, install `kubewarden-defaults`.
+
+On Rancher Fleet, one can codify the chart dependencies using [`dependsOn` in the _fleet.yaml_ file](https://fleet.rancher.io/gitrepo-structure#fleetyaml).
```

Comment:
```suggestion
On Rancher Fleet, one can codify the chart dependencies using `dependsOn` in the [_fleet.yaml_ file](https://fleet.rancher.io/gitrepo-structure#fleetyaml).
```

Reply:
Done in https://github.com/kubewarden/docs/pull/161/commits/b4321ddca87846628677ab41c40917f8b608bb21, suggestion was marked as outaded.